### DISCUSSION TITLE
11월 14일 주간회의 쓰레드

### DISCUSSION THREAD

**Discussion by euijinkk** [woowacourse/prolog#1117] [General]
11월 14일 주간회의 쓰레드

**Comment by euijinkk** [comment]
## 로드맵 어드민 페이지 구상 및 구현 중 (with @wonsss)

- react-admin을 활용하여 custom page 생성 방법 공부
- Roadmap 어드민 페이지 구상
- Roadmap 어드민 페이지 개발 진행 중

react-admin을 사용하면서 커스텀이 필요한 영역은 따로 구현하기로 함 (앨버, 동키콩, 마르코, 시지프 회의)

**Comment by kwannee** [comment]
- wiki에 프론트엔드 온보딩 문서 작성
  - 폴더 구조
  - 코드 스타일 가이드
  - 디자인 시스템 
- 프론트엔드 크루들과 토의를 통해 추가적인 컨벤션 작성 예정
